Ollivanders/src/net/pottercraft/Ollivanders2/Spell/IncendioSuper.java

```diff
@@ -99,7 +98,7 @@ public void checkEffect ()
          }
          for (SpellProjectile proj : p.getProjectiles())
          {
-            if ((proj.name == Spells.GLACIUS || proj.name == Spells.GLACIUS_DUO || proj.name == Spells.GLACIUS_TRIA)
+            if ((proj.spellType == O2SpellType.GLACIUS || proj.spellType == O2SpellType.GLACIUS_DUO || proj.spellType == O2SpellType.GLACIUS_TRIA)
```

Comment:
did we want to change name here?

Reply:
Yes, since the name of the class changed and "name" is used in too many places.